File: php/raindrops/raindrops.php
<?php

function raindrops(int $num) : string
{
  $str = '';
  if ($num % 3 === 0) {
    $str .= 'Pling';
  }
  if ($num % 5 === 0) {
    $str .= 'Plang';
  }
  if ($num % 7 === 0) {
    $str .= 'Plong';
  }
  if ($str === '') {
    $str = $num;
  }
  return $str;
};

File: go/raindrops/raindrops.go
package raindrops

import "fmt"

func Convert(num int) string {
	str := ""
	if num%3 == 0 {
		str += "Pling"
	}
	if num%5 == 0 {
		str += "Plang"
	}
	if num%7 == 0 {
		str += "Plong"
	}
	if str == "" {
		str = fmt.Sprintf("%d", num)
	}
	return str
}
